releasenotes-builder/src/main/java/com/github/checkstyle/CliProcessor.java

```diff
@@ -296,6 +300,8 @@ private static Options buildOptions() {
             "Access token secret for Twitter.");
         options.addOption(OPTION_TWITTER_PROPERTIES, true,
             "Properties for publication on Twitter.");
+        options.addOption(OPTION_VALIDATE_VERSION, false,
+            "Whether pom version should be validated in issues labels.");
         return options;
```

Comment:
We kind of do not care about pom,
We validate `options.addOption(OPTION_RELEASE_NUMBER, true, "Release number.")`

Reply:
Is `Whether release notes labels should be validated based on release number.` better?

Reply:
Whether release notes labels should be validated based on release number.

Reply:
Done.